.gitlab-ci.yml

```diff
@@ -534,6 +534,30 @@ test-suite:edge:dune:dev:
     # Gitlab doesn't support yet "expire_in: never" so we use the instance default
     # expire_in: never
 
+test-suite:edge+4.11+trunk+dune:
+  stage: stage-1
+  dependencies: []
+  script:
+    - opam switch create 4.11.0 --empty
```

Comment:
Since we use a fresh switch should we use a different image?

Reply:
Not sure I'd be worth to keep our own image just for this; unless we have already some image with all the Debian packages that Coq needs installed etc... do you know of any candidate?

Reply:
I've looked into it and did not find any Docker image with opam installed but no switch created.